android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormPresenter.kt

```diff
@@ -44,6 +44,13 @@ class DDMFormPresenter(val view: DDMFormViewContract.DDMFormView) : DDMFormViewC
 			val isValid = fieldContext.isValid ?: true
 
 			field.lastValidationResult = isValid
+
+			if (!isValid) {
```

Comment:
Maybe invert the order is better. `if (isValid) else`

Reply:
Yeah, that's true.

Reply:
Fixed